hazelcast/include/hazelcast/client/reliable_topic.h

```diff
@@ -156,6 +159,7 @@ class HAZELCAST_API reliable_topic : public proxy::ProxyImpl
           , cancelled_(false)
           , logger_(lg)
           , name_(topic_name)
+          , execution_service_(execution_service)
```

Comment:
`execution_service_` instance will be copy-constructed rather than move constructed. So I suggest to change it as :

```suggestion
          , execution_service_(move(execution_service))
```

Reply:
Fixed